templates/misc/message_form.html

```diff
@@ -23,6 +23,10 @@ <h4 class="alert-box-title">Pas encore inscrit ?</h4>
             <a href="{% url "register-member" %}" class="alert-box-btn">{% trans "Créer un compte" %}</a>
         </p>
     </div>
+{% elif not user.can_write_now %}
+<div class="alert-box ico-after ico-alert light">
+    {% trans "Vous êtes en lecture seule. Vous ne pouvez pas poster de messages." %}
+</div>
```

Comment:
Il serait mieux de respecter l'indentation ici

```suggestion
{% elif not user.can_write_now %}
    <div class="alert-box ico-after ico-alert light">
        {% trans "Vous êtes en lecture seule. Vous ne pouvez pas poster de messages." %}
    </div>
```